openlibrary/plugins/openlibrary/js/merge-request-table/index.js

```diff
@@ -71,17 +72,25 @@ export function initCommenting(elems) {
  * @param {HTMLTextAreaElement} textarea The element that contains the comment
  * @param {Number} mrid Unique identifier for the request that is being commented on
  */
-async function onCommentClick(textarea, mrid) {
+async function onCommentClick(textarea, mrid, username) {
     const c = textarea.value;
+<<<<<<< HEAD
+    const commentCount = document.querySelector('.comment-count');
+
+=======
+    const commentCount = document.querySelector(`.comment-count-${mrid}`);
+
+>>>>>>> e32ef5fc7 (fix bug where button is not working)
```

Comment:
There seems to be a merge conflict checked in here

Reply:
I merged the new changes!